versioned_docs/version-8.5/apis-tools/spring-zeebe-sdk/getting-started.md

```diff
@@ -124,6 +124,13 @@ ZEEBE_CLIENT_ID=xxx
 ZEEBE_CLIENT_SECRET=xxx
 ```
 
+Example environment variables to be set to configure gRPC and REST connection:
+
+```properties
+ZEEBE_GRPC_ADDRESS=http://127.0.0.1:26500/
+ZEEBE_REST_ADDRESS=http://127.0.0.1:8083/
```

Comment:
This looks like the path for Tasklist, but in the 8.5 docs I see `8080` for the [Zeebe API](https://docs.camunda.io/docs/8.5/apis-tools/zeebe-api-rest/zeebe-api-rest-overview/). Can you help me understand this change?

Reply:
I will make the change now.